int: current_sector
float: current_sector_time
float: current_sector_mean_speed
